### DISCUSSION TITLE
Pimcore with Docker under WSL2

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#10222] [Q&A / Support]
I am not a docker professional and would like to know if I understood correctly how to install Pimcore with Docker and WSL2.

- I install Docker-Desktop
- I install a linux distro under WSL2
- I install Composer (and therefore PHP 8) in the distro
- I run Composer to install Pimcore Skeleton
- I go to the Pimcore folder and I run docker-compose as shown [here](https://github.com/pimcore/skeleton)

Is that true?

If it is, what is the advantage of Docker if I have to install PHP 8 (and therefore Apache...)?

Sorry if my question seems stupid to you, I'm just trying to understand

**Comment by Danmir666** [comment]
Hello, you don't need to install composer to run the docker images, you just need Docker and Docker-compose.

You can just follow the instructions of the demo package [Demo](https://github.com/pimcore/demo#follow-these-steps), it will create a new folder with the demo project.

In case your docker is not configured to run as root, you can edit your docker-compose.yml, and add in each image **privileged: true** to test it. Change it when you deploy to prod.

It would be like this

`image: redis:alpine
    privileged: true`

for each image in the docker-compose
